index.bs

```diff
@@ -179,20 +181,19 @@ interface mixin UnidirectionalStreamsTransport {
          1. The |transport|'s state transitions to `"closed"` or `"failed"`.
          1. |p| has not been [=settled=].
 
-: <dfn for="UnidirectionalStreamsTransport" method>receiveStreams()</dfn>
-:: Returns a {{ReadableStream}} of {{ReceiveStream}}s that have been received
-   from the remote host.
+: <dfn for="UnidirectionalStreamsTransport" attribute>incomingUnidirectionalStreams</dfn>
+:: A {{ReadableStream}} of unidirectional streams, each represented by a
+   {{ReceiveStream}} object, that have been received from the remote host."
 
-   When `receiveStreams` is called, the user agent MUST run the following
-   steps:
+   The getter steps for `incomingUnidirectionalStreams` are:
      1. Let |transport| be the `UnidirectionalStreamsTransport` on which
-        `receiveStreams` is invoked.
+        `incomingUnidirectionalStreams` getter is invoked.
```

Comment:
Maybe it should be **the** \`incomingUnidirectionalStreams\` getter?

Reply:
Fixed.